src/PropertySet.cc

```diff
@@ -202,12 +203,12 @@ T PropertySet::get(std::string const& name)
         throw LSST_EXCEPT(pex::exceptions::NotFoundError, name + " not found");
     }
     try {
-        return boost::any_cast<T>(i->second->back());
-    } catch (boost::bad_any_cast) {
+        return std::any_cast<T>(i->second->back());
+    } catch (std::bad_any_cast &) {
```

Comment:
```suggestion
    } catch (std::bad_any_cast const&) {
```

Same with other cases below.